app/src/main/res/values/dimens.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <dimen name="sizeTextHeader">20dp</dimen>
-    <dimen name="sizeTextHeader2">15dp</dimen>
-    <dimen name="marginBottomHorizontalScrollView">10dp</dimen>
-    <dimen name="marginBetween2Item">5dp</dimen>
+    <dimen name="sizeTextHeader">15sp</dimen>
```

Comment:
pls check convention.
Resource IDs and names need to be declared in lowercase_underscore